plugins/ExtendedTools/ExtendedTools.vcxproj

```diff
@@ -83,8 +83,8 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <Link>
-      <AdditionalDependencies>cfgmgr32.lib;pdh.lib;tbs.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <DelayLoadDLLs>cfgmgr32.dll;oleaut32.dll;tdh.dll;tbs.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
+      <AdditionalDependencies>cfgmgr32.lib;pdh.lib;tbs.lib;winsta.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <DelayLoadDLLs>cfgmgr32.dll;oleaut32.dll;tdh.dll;tbs.dll;winsta.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

Comment:
Your build is failing since this need copied to the other platforms and configurations.

Reply:
Oops, I completely forgot to update other configurations.
f93b8c5cf0db94c7745a9ab59d062e8f7481514a